config/initializers/session_store.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.session_store :cache_store
```

Comment:
Why has this been added?

We should be using the cookie_store. Otherwise, we will need to introduce a cache (like Redis) somewhere.